src/main/java/de/tum/in/www1/artemis/domain/User.java

```diff
@@ -460,4 +464,9 @@ public Set<DataExport> getDataExports() {
     public void setDataExports(Set<DataExport> dataExports) {
         this.dataExports = dataExports;
     }
+
+    @Nullable
+    public ZonedDateTime isIrisAccepted() {
```

Comment:
since it's a timestamp I think we should rename this method to `irisAcceptedAt`, probably change column name and field name too if you want